### DISCUSSION TITLE
Error get user with API token

### DISCUSSION THREAD

**Discussion by brunodsazevedo** [adonisjs/core#2592] [Help] [answered]
Hello. I'm having trouble creating a store profile method, where to register a new profile, the user needs to be logged in. So I'm using:

 `auth.use ('api').user` and `auth.use('api).authenticate()`. However, my return has an error in the token. what did I do wrong?
 
 ```public async store ({request, response, auth}: HttpContextContract) {
    const user = await auth.use('api').authenticate()
    console.log(auth.use('api').user)
    const data = request.only([
      'first_name',
      'last_name',
      'cpf',
      'address',
      'address_number',
      'address_detail',
      'neighborhood',
      'city',
      'state',
      'zip_code',
      'phone',
      'cellphone',
      'whatsapp',
    ])

    const profile = await Profile.create({...data, user_id: user.id})
    return response.status(200).send({user})
  }```

```{
  "errors": [
    {
      "message": "E_INVALID_API_TOKEN: Invalid API token"
    }
  ]
}```

PS: I using Insomnia for test API and i setting Bearer token.

**Comment by RomainLanz** [comment, marked as answer]
Hey @brunodsazevedo! 👋 

How do you send your API token?

**Reply by brunodsazevedo** [reply]
I  send Bearer token with insomnia like that:
![Screenshot_20210512_091042](https://user-images.githubusercontent.com/58368716/117972880-0fb8ae00-b302-11eb-8aee-fd396e1b417e.png)

I try send header with Authorization, but returns same error

**Reply by thetutlage** [reply]
Can you please a repo for us to reproduce the issue?

**Reply by brunodsazevedo** [reply]
Shure. I created a clone repo public.
https://github.com/brunodsazevedo/clone-repo-adonisv5-for-issue-method-auth

**Reply by thetutlage** [reply]
I see you are using a very small expiry duration. `3600` milliseconds is just `3.6` seconds. I recommend using a string based value to keep it more clear. Examples here https://docs.adonisjs.com/guides/auth/api-tokens-guard#manging-tokens-expiry

**Reply by brunodsazevedo** [reply]
OMG.  I was really thinking that the time was in seconds instead of milliseconds.  No problem, I will then follow the example of the documentation and test again.

**Reply by brunodsazevedo** [reply]
It's worked. Thanks!
![Screenshot_20210514_005828](https://user-images.githubusercontent.com/58368716/118219432-d7bf8100-b44f-11eb-9524-2d68070b41b4.png)
